src/com/metricsassignment/metrics/LCOMMetric.java

```diff
@@ -20,41 +20,37 @@
     @Override
     public double calculate(CompilationUnit compilationUnit) {
     	p = 0; q = 0;
-    	List<Set<String>> fieldsSetList = new ArrayList<Set<String>>();
+    	List<Set<String>> fieldsSetList = new ArrayList<>();
```

Comment:
Why change this?

Reply:
It's just redundant.

Reply:
I believe its required by the syntax. I am not sure if it causes any effects.

Reply:
To me it shows up as redundant and IntelliJ throws warnings all over the place. No preference really.